config/7.1.0/securebanking/ig/scripts/groovy/ProcessRegistration.groovy

```diff
@@ -302,3 +307,38 @@ private boolean validateRegistrationJwtSignature(jwt, jwkSet) {
         return false
     }
 }
+
```

Comment:
Method comment with link to relevant specs might be useful for those following....

Reply:
Done